opnode/bss/package.json

```diff
@@ -0,0 +1,6 @@
+{
+	"name": "@eth-optimism/batch-submitter-service",
```

Comment:
Do we have docs on this/the changelog?

Reply:
I've removed the changelog for now, we can add it at a later time if we want